Resources/Prototypes/SimpleStation14/Entities/Clothing/Eyes/glasses.yml

```diff
@@ -0,0 +1,18 @@
+- type: entity
+  parent: ClothingEyesBase
```

Comment:
Parent from Secglasses, and remove the redundant components. Just add the prescription glasses thing and the drop chance.